.storybook/main.js

```diff
@@ -5,6 +5,7 @@ const config = {
     '@storybook/addon-links',
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
+    'storybook-addon-sass-postcss',
```

Comment:
Another PR is already adding this, seems like we actually need this hmm.

Reply:
Yup

Reply:
@manishprivet Were you able to process sass files for story without this addon?